<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sample</class>
 <widget class="qMRMLWidget" name="Sample">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>527</width>
    <height>257</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="ctkCollapsibleButton" name="inputsCollapsibleButton">
     <property name="text">
      <string>Inputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Input Directory with DICOMs: </string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkDirectoryButton" name="inDirButton">
        <property name="toolTip">
         <string>Select directory with DICOMs</string>
        </property>
        <property name="toolTipDuration">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="inDetailsLabel">
        <property name="text">
         <string>No directory selected</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="ctkCollapsibleButton" name="outputsCollapsibleButton">
     <property name="text">
      <string>Outputs</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output directory:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ctkDirectoryButton" name="outDirButton">
        <property name="toolTip">
         <string>Select a directory to copy the anonymized files to.</string>
        </property>
        <property name="toolTipDuration">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="useUUIDCheckBox">
        <property name="toolTip">
         <string>Use UUID to rename the files. Prefixes will be ignored if this is enabled.</string>
        </property>
        <property name="toolTipDuration">
         <number>5</number>
        </property>
        <property name="text">
         <string>Use UUID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="prefixLineEdit">
        <property name="toolTip">
         <string>Files will be renamed with this prefix, if UUID is not requested</string>
        </property>
        <property name="toolTipDuration">
         <number>5</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Run deanonymization</string>
     </property>
     <property name="toolTipDuration">
      <number>5</number>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkDirectoryButton</class>
   <extends>QWidget</extends>
   <header>ctkDirectoryButton.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
